src/shared/clock-util.c

```diff
@@ -137,12 +137,15 @@ int clock_reset_timewarp(void) {
         return 0;
 }
 
-#define EPOCH_FILE "/usr/lib/clock-epoch"
+#define EPOCH_FILE "/var/lib/systemd/clock-epoch"
+#define DELTA_THRESHOLD (USEC_PER_YEAR*8)
```

Comment:
this will break long running systems - ie, the support window for the civil infranstructure lts is 15 years

Reply:
please make this configurable via meson param. and better pick a value too large than too short.